### ISSUE TITLE
JDK 11: cglib warning when building

### ISSUE THREAD

**Issue by barthanssens** [eclipse/rdf4j#1423]
On ARMv64, dev branch, maven 3.6.0, probably maven related

```
Java version: 11.0.4, vendor: Debian, runtime: /usr/lib/jvm/java-11-openjdk-arm64
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "4.15.11-mainline-rev1", arch: "aarch64", family: "unix"
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.google.inject.internal.cglib.core.$ReflectUtils$1 (file:/usr/share/maven/lib/guice.jar) to method java.lang.ClassLoader.defineClass(java.lang.String,byte[],int,int,java.security.ProtectionDomain)
WARNING: Please consider reporting this to the maintainers of com.google.inject.internal.cglib.core.$ReflectUtils$1
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

**Comment by barthanssens** [comment]
Related to #1148

**Comment by barthanssens** [comment]
Fixed in maven 3.6.1
